less/bootstrap-override/buttons.less

```diff
@@ -104,6 +102,56 @@
 	}
 }
 
+.btn-basic {
+	//because this button has 1px border, padding needs to be adjusted
+	//padding + border = 5px
+	padding: 4px 10px;
+	border: 1px solid #bbb;
+	#gradient > .gradient-bg(#FEFEFE,#E0E0E0);
+
+	&:active,&.active,&:focus{
+		background: @gray93;
+		.box-shadow(inset 0 1px 1px rgba(0,0,0,.2));
+	}
+
+
+	border: 1px solid @btn-basic-border;
+	.button-size(4px; 10px; @font-size-base; @border-radius-base);
+	.repeater-views & {
+		.button-size(3px; 5px; @font-size-base; @border-radius-base);
+		.glyphicon, .fuelux-icon {
+			width: 11px;
+			height: 11px;
+		}
+	}
+
+	.button-variant(@btn-basic-color;
+									@btn-basic-bg-start; @btn-basic-bg-end;
+									@btn-basic-bg-hover-start; @btn-basic-bg-hover-end;
+									@btn-basic-bg-press-start; @btn-basic-bg-press-end;
+									@btn-basic-bg-disabled-start; @btn-basic-bg-disabled-end);
+	&:active,&.active,&:focus,&:active:focus,&:active:hover {
+		#gradient > .gradient-bg(@btn-basic-bg-press-start, @btn-basic-bg-press-end);
+		color: @btn-basic-color;
+	}
+
+	&.dropdown-toggle .caret {
+		color: @btn-basic-color;
+	}
+
+	&.btn-lg {
```

Comment:
can the `.btn-lg` `.btn-sm` `.btn-xs` portions be dried up? is the padding different here than on other button types?